src/index.tsx

```diff
@@ -6,6 +6,9 @@ import './index.css';
 import config from './config';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers'
```

Comment:
Should put 3rd party imports first then local imports separated by a new line

Reply:
Thanks. Update import order.